config/config.go

```diff
@@ -60,17 +63,21 @@ var (
 	DefaultRegexpExtract = RegexpExtract{
 		Value: "$1",
 	}
+
+	ErrUnsupportedVersion = errors.New("unsupported config version")
```

Comment:
```suggestion
	ErrUnsupportedVersion = errors.New("Unsupported config version, we only support config version 2. Please see https://github.com/prometheus/snmp_exporter/blob/main/README.md#debugging")
```

Reply:
This is a Go error, it shouldn't be this verbose.

Reply:
Will the user see it on CLI?

Reply:
Yes, but basically no. That's the whole reason for the "Possible old config file" error. The user won't directly see this error. It's for internal `errors.Is()` use.